.travis/install-petsc.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-git clone https://bitbucket.org/petsc/petsc.git
+git clone https://gitlab.com/petsc/petsc.git
```

Comment:
A number of these commits look the same as in #26, did I just take too long to merge and thus they are showing up?

Reply:
I will rebase this branch of the latest master.